src/main/java/stream/flarebot/flarebot/commands/moderation/PermissionsCommand.java

```diff
@@ -145,6 +143,19 @@ public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Mess
                         group.getPermissions().clear();
                         MessageUtils.sendSuccessMessage("Cleared all permissions from the group: " + group.getName(), channel);
                         return;
+                    } else if (args[2].equalsIgnoreCase("move")) {
+                        if(args.length >= 4) {
+                            int pos = GeneralUtils.getInt(args[3], -1);
+                            if(pos < 1 || pos > (guild.getPermissions().getGroups().size() - 1)) {
+                                MessageUtils.sendErrorMessage("Invalid Page: " + args[3], channel);
```

Comment:
Can we please stop using sendErrorMessage for things that are not actually an error. Use sendWarningMessage